lib/Elastica/Cluster.php

```diff
@@ -138,8 +127,6 @@ public function getClient()
      * @see https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-nodes-info.html
      *
      * @param array $args Additional arguments
-     *
-     * @throws \Elastica\Exception\NotImplementedException
```

Comment:
Isn't that still needed? Or is this replaced by some "magic"?

Reply:
This method is just a placeholder, as it is only throwing that exception.
I was in doubt to remove the method completely ... wdyt?

Reply:
I think my "idea" here was that if users see an method and get an exception with "not implemented" they are likely to contribute it. But seems like it never happened. I would say either we keep the docs entry or remove the method completly. I'm good with removing it completely. I guess there are 2-3 other similar methods.

Reply:
Removed the "not-implemented" code!